Dockerfile

```diff
@@ -11,8 +11,8 @@ RUN apt-get update \
     && apt-get -y install sudo python3-pip \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 RUN pip install --upgrade pip;\
-    pip install pipenv;\
-    PIP_IGNORE_INSTALLED=1 pipenv install --system --deploy;\
+    pip install poetry;\
+    PIP_IGNORE_INSTALLED=1 poetry install;\
```

Comment:
@markusressel I haven't used the Dockerfile yet. Is it from your perspective simple as that replacing `pipenv` with `poetry` in the Dockerfile?

Reply:
No. Take a look at this one: https://github.com/ekeih/alerticular/blob/main/Dockerfile

Reply:
The referenced Dockerfile uses Alpine as a base, which has its downsides, see: https://pythonspeed.com/articles/alpine-docker-python/

For a version based on buster, have a look at this one:
https://github.com/markusressel/openhasp-config-manager/blob/main/Dockerfile